src/postgres/tree/PostgresColumnTreeItem.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { AzureTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { PostgresTableTreeItem } from './PostgresTableTreeItem';
+
+export class PostgresColumnTreeItem extends AzureTreeItem<ISubscriptionContext> {
+    public static contextValue: string = "postgresColumn";
+    public readonly contextValue: string = PostgresColumnTreeItem.contextValue;
+    public readonly columnName: string;
+    public readonly parent: PostgresTableTreeItem;
+
+    constructor(parent: PostgresTableTreeItem, columnName: string) {
+        super(parent);
+        this.columnName = columnName;
+    }
+
+    public get id(): string {
+        return String(this.columnName);
+    }
+
+    public get label(): string {
+        return this.columnName;
+    }
+
+    public get iconPath(): string | vscode.Uri | { light: string | vscode.Uri; dark: string | vscode.Uri } {
+        return getThemeAgnosticIconPath('PostgresColumn.svg');
```

Comment:
Please post screenshots of the tree in both light and dark theme with your new icons

Reply:
![image](https://user-images.githubusercontent.com/17537419/81026565-63f79b00-8e2f-11ea-8b9c-274d3e5a2ebe.png)
![image](https://user-images.githubusercontent.com/17537419/81027503-d61daf00-8e32-11ea-9823-b6513e9466e2.png)
Icons are not visible with Abyss and Dark theme. 
Can we modify these icons or have custom icons or do we have to specifically use icons from https://cds-icons.azurewebsites.net/ ?

Reply:
Well you can also try the VS Code icon set, but I didn't see any table stuff on a quick glance: https://github.com/Microsoft/vscode-icons

We definitely need something that works with default light and default dark, so maybe delay the icon part of this PR. Personally, I think the Azure ones you found look pretty ugly anyways haha. I might be making this up, but I thought Matt he said he'd help with icons - so maybe ask him?